docs/core/tools/csproj.md

```diff
@@ -33,10 +33,10 @@ Metapackages are implicitly referenced based on the target framework(s) specifie
 ### Recommendations
 Since `Microsoft.NETCore.App` or `NetStandard.Library` metapackages are implicitly referenced, the following are our recommended best practices:
 
-* Never have an explicit reference to the `Microsoft.NETCore.App` or `NetStandard.Library` metapackages via a `<PackageReference>` item in your project file.
-* If you need a specific version of the runtime, you should use the `<RuntimeFrameworkVersion>` property in your project (for example, `1.0.4`) instead of referencing the metapackage.
+* Never have an explicit reference to the `Microsoft.NETCore.App` or `NetStandard.Library` metapackages via a `<PackageReference>` item in your project file when trageting .NET Core or .NET Standard.
```

Comment:
typo: tagreting
I'd also start with the condition: When targeting ..., never have ...

Also wondering if we want to be more explicit about the .NET Framework case here too?

Reply:
Thanks!